BLaurenB/techtime
# Add quantity to order show pages - user & admin

#### What's this PR do?
Adds the quantity and price in the correct columns when viewing order details (order show).

#### Where should the reviewer start- what are the pertinent changes and the sequence?
app/views/orders/show.html.erb
app/views/admin/orders/show.html.erb

#### How should this be manually tested?
place an order and then view the order. it should have the quantity and price on the page.

#### Any background context you want to provide?
#### What are the relevant tickets?
Closes #115 
#### Screenshots (if appropriate)
#### Questions:
